contributing_to_docs/term_glossary.adoc

```diff
@@ -352,6 +352,16 @@ Usage: Kubernetes API server
 
 == M
 
+''''
+=== MetalLB
+
+Usage: MetalLB, MetalLB Operator, MetalLB project
+
+MetalLB is an open source project that provides a way to add services of type `LoadBalancer` to clusters that are not installed on infrastructure from a cloud provider.
+The primarily targeted infrastructure is on-premise, bare-metal clusters, but any infrastructure that does not include a native load-balancing capability is a candidate.
+
+"MetalLB" is always capitalized as shown in general text.
```

Comment:
Thinking of alternative ways to phrase this so that it's explicit: 

- `"MetalLB" always has the first letter and last two letters capitalized in normal text.`
- `In normal text, always capitalize "MetalLB" in its stylized form. Do not write "metallb," "metalLB," or "Metallb."`